ui/core.js

```diff
@@ -13,6 +13,8 @@
 //>>description: The core of jQuery UI, required for all interactions and widgets.
 //>>docs: http://api.jqueryui.com/category/ui-core/
 //>>demos: http://jqueryui.com/
+//>>css.structure: ../themes/base/core.css
```

Comment:
Ok so I looked into this more to figure out how exactly this is happening in mobile since it was non obvious. Some of the modules that make up our `core.js` actually depend on `core.css`. So in mobile we are essentially doing exactly as @rxaviers had originally suggested. However that aside I still don't think thats what we should do here. 

There are 3 ways i could see us handling this.
1) We could explicitly make any files that actually depend on it, depend on it explicitly ( list it in all the widgets under structure )
2) Since under the hood this is `r.js`, we can just shim it, to make ever other css file depend on it.
3) we could look at allowing dependency comments in the css files as well. Then add the dependency directly to the css files.